docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -24,13 +24,39 @@ embedded assets. This can be different for older releases.
 
 ## Download
 
-Choose the file matching your platform from the [downloads page](https://dl.gitea.io/gitea/), copy the URL and replace the URL within the commands below:
+You can find the file matching your platform from the [downloads page](https://dl.gitea.io/gitea/) after navigating to the version you want to download. 
+
+### Choosing the right file
+For Linux, you should choose: 
+ * `linux-386` for 32-bit Intel/AMD processors
+ * `linux-amd64` for 64-bit Intel/AMD processors
+ * 'linux-arm64' for 64-bit ARM systems like a Raspberry PI 4
+ * `linux-arm-5` and `linux-arm-6` are for other versions of the ARM processor 
+ 
+For Windows, you should chose:
+ * `windows-4.0-386` for 32-bit Intel/AMD processors
+ * `windows-4.0-amd64` for 64-bit Intel/AMD processors
+ * `gogit-windows` if you want to use the built-in (Go-based) git client; otherwise you will need to install Git.
```

Comment:
Looks that's not right, `go-git` still need git command to do something like `merge`..
the reason of add `go-git` is here : https://github.com/go-gitea/gitea/pull/15482